modules/apps/site/site-cms-site-initializer/test/js/main_view/bulk_action_task/util/index.test.tsx

```diff
@@ -13,7 +13,7 @@ import {URL_BULK_ACTION_TASK} from '../../../../../src/main/resources/META-INF/r
 
 const OBJECT_ENTRY_FOLDER_CLASS_NAME =
 	'com.liferay.object.model.ObjectEntryFolder';
-describe('Bulk Actions Monitor Utils', () => {
+describe.skip('Bulk Actions Monitor Utils', () => {
```

Comment:
an old .skip or is it correct?

Reply:
I've agreed this with Gianmarco, he will restore the tests with his PR